sdk/resourcemanager/azure-resourcemanager-storage/src/test/java/com/azure/resourcemanager/storage/StorageAccountOperationsTests.java

```diff
@@ -221,6 +221,12 @@ public void canDisallowCrossTenantReplicationOnStorageAccount() {
                 .withSku(StorageAccountSkuType.STANDARD_LRS)
                 .create();
 
+        Assertions.assertFalse(storageAccount.isAllowCrossTenantReplication());
+
+        storageAccount.update()
+            .allowCrossTenantReplication()
+            .apply();
+
```

Comment:
Thanks. Should we confirm with service about this behavioral change? Guess if it's expected, we should put that in breaking change section.

Reply:
When creating a Storage Account on the portal, the default value of this parameter is `false`.
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/21334f36-7518-4800-b558-e3ccfd88b3d8)

If it is a breaking change, the `allowCrossTenantReplication` interface must be added in `Create` mode.

Reply:
It appears being changed due to security reason. Please document it in breaking changes
https://learn.microsoft.com/en-us/rest/api/storagerp/storage-accounts/create?view=rest-storagerp-2023-01-01

Reply:
You can merge this PR first, and immediately make a follow-up PR to add `allowCrossTenantReplication` API to create flow.
https://github.com/azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/StorageAccount.java#L675

Reply:
So do we need to fix `Create` mode?

Reply:
Yes, in a follow-up PR.